README.md

```diff
@@ -119,7 +49,7 @@ To end your Jupyter session, press `Ctrl-c`. This will prompt you to `Shutdown t
 
 To deactivate your virtualenv, simply run `deactivate` from the terminal window where you activated `openvino_env`. This will deactivate your environment.
 
-To reactivate your environment, simply repeat [Step 3](#step-3-activate-the-environment) from the Install Guide. 
+To reactivate your environment, simply repeat [Step 3](#step-3-activate-the-environment) from the Install Guide.
```

Comment:
It is not required to reactivate the virtual environment to run the notebooks. Should we explain this? You need to reactivate the environment if you want to install new/updated packages. But just to run the notebooks, you can just do `jupyter lab notebooks` again.